webui/react/src/pages/ProjectDetails.tsx

```diff
@@ -1091,6 +1092,8 @@ const ProjectDetails: React.FC = () => {
   if (isNaN(id)) {
     return <Message title={`Invalid Project ID ${projectId}`} />;
   } else if (!expPermissions.canViewWorkspaces) {
+    // If user does not have access to the default workspace, then redirect to the workspace list page, as post-login landing page.
+    if (project?.id === 1) navigate(paths.workspaceList(), { replace: true });
```

Comment:
Instead of double-redirecting users, we should just redirect them to the workspace list right away [from SignIn](https://github.com/determined-ai/determined/blob/master/webui/react/src/pages/SignIn.tsx#L77). 

We'll want to have the redirect wait for roles to be available (if rbac is on).

Reply:
Sure no problem, post login page changed